modules/tagging/README.md

```diff
@@ -0,0 +1,36 @@
+# Resource Tagging Module
```

Comment:
Include following functionalities

1) Metadata file
2) Detect secret
3) update test directory to execute resource tagging templates
4) Root module

Make it as Type1 module

Reply:
Added below files:

Metadata file
Detect secret
update test directory to execute resource tagging templates
Root module